Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -191,7 +225,7 @@ class Testball < Formula
   ensure
     cmd("uninstall", "--force", "testball")
     cmd("cleanup", "--force", "--prune=all")
-    formula_file.unlink unless formula_file.nil?
+    @formula_file.unlink unless @formula_file.nil?
```

Comment:
I reckon you could pull the repeated `.unlink`s into methods too to save leaking the `@formula_file` into a method and just to be more consistent, even the same number of lines of code.